lib/Hero/Hero.js

```diff
@@ -0,0 +1,122 @@
+import React, { lazy, Suspense } from 'react';
+import PropTypes, { element, elementType, node } from 'prop-types';
+import { Button as ButtonComponent } from '../Button';
+import Content from './Content';
+const Carousel = lazy(() =>
```

Comment:
Good to lazy load the carousel. Is there any way we could conditionally load it only if we need it (`slides.length > 1`)?

Reply:
I couldn't find a good way to lazy load the component

Reply:
Yes, we'd need Babel in the picture to perform dynamic imports. Nevermind!